tests/test_performance_limits.py

```
@@ -0,0 +1,402 @@
+        """Create JsonTableDirective instance for testing."""
+        # Mock the required arguments for SphinxDirective
+        name = "jsontable"
+        arguments = []
+        options = {}
+        content = []
+        lineno = 1
+        content_offset = 0
+        block_text = ""
+        state = MagicMock()
+        state_machine = MagicMock()
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)